fullnode/src/main.rs

```diff
@@ -33,6 +33,13 @@ fn main() {
                 .takes_value(true)
                 .help("File containing an identity (keypair)"),
         )
+        .arg(
+            Arg::with_name("staker_keypair")
+                .long("staker-keypair")
+                .value_name("PATH")
+                .takes_value(true)
+                .help("File containing the staker's keypair"),
```

Comment:
How about adding more help info like "Note- The staker's account must be setup and delegated to this node beforehand" ?

Reply:
This is a quick fix to get Beacons out the door.  We can delete it once vote_account_id is plumbed through to ReplayStage.

Reply:
Just to clarify, passing the staking keypair into fullnode is a bad thing to do. Nobody with real stake should ever do it.